proto/config/devconfig.proto

```diff
@@ -169,6 +169,14 @@ message EdgeDevConfig {
 
   // cluster configuration
   EdgeNodeCluster cluster = 43;
+  DhcpVendorExtensionsOverride dhcp_extensions_override = 44;
```

Comment:
I think this should be part of `NetworkConfig` instead. Either inside `ipspec` or probaly better as another field.

Reply:
You can have several `NetworkConfig`s - what is the meaning if you set it in one to `true` and in another one to `false`?
That means that for network device eth0 you can override the dhcp option and for another network device eth1 you don't override the dhcp option.
Is my interpretation correct?

Reply:
Yes, that would be the preferred behavior.